packages/contracts-bedrock/src/L1/SystemConfigInterop.sol

```diff
@@ -5,7 +5,7 @@ import { Constants } from "src/libraries/Constants.sol";
 import { OptimismPortalInterop as OptimismPortal } from "src/L1/OptimismPortalInterop.sol";
 import { GasPayingToken } from "src/libraries/GasPayingToken.sol";
 import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import { SystemConfig } from "src/L1/SystemConfig.sol";
+import { SystemConfig, ResourceMetering, Storage } from "src/L1/SystemConfig.sol";
```

Comment:
nit: import `Storage` from the library where it is defined rather than from `SystemConfig` ?